### PR TITLE
Added tutorial for using rbxlx-to-rojo 

### LINKED ISSUE #224: Since rbxlx-to-rojo now works, update existing-game.md to have a tutorial on using the tool
Title says it all.
The tool is now fully functional, so, [existing-game.md ](https://github.com/rojo-rbx/rojo/blob/master/docs/guide/existing-game.md) should be updated to have a little tutorial on how to use it.

### PR DESCRIPTION
**Pull request by Jaguar-515**
Fixes #224.  
(yes, i know the issue and this pr is made by me, i'm just doing this because nobody has made a fix for the issue up until now.)

Keep in mind this is an draft, and stuff might be changed in the future.

### CODE REVIEW

File: docs/guide/existing-game.md [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
-    * [rbxlx-to-rojo](https://github.com/rojo-rbx/rbxlx-to-rojo)
-    * `rojo export` ([issue #208](https://github.com/rojo-rbx/rojo/issues/208))
-* Can port as much or as little of your game as you like
-    * Rojo can manage just a slice of your game!
-* Some Roblox idioms aren't very well supported
-    * Redundant copies of scripts don't work well with files
-    * Having only a couple places with scripts simplifies your project dramatically!
+# Using rbxlx-to-rojo
+## Setup
+Before you can use rbxlx-to-rojo, you need to have Rojo 0.5.0 Alpha 12 to use the tool.
```
Comment (Kampfkarren) [inline docs/guide/existing-game.md]: > you need to have Rojo 0.5.0 Alpha 12 
You need *at least* alpha 12

File: docs/guide/existing-game.md [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
+## Porting the game
+Let's assume we have a rbxlx file named "game", with a script in ServerScriptService.
+
+### Steps to port the game:
+1. Double-click on rbxlx-to-rojo on wherever you installed it.
+2. Select a place file.
+3. Select the path where you want to place it.
+
+If you followed the steps correctly, you should see something that looks like this:
+https://cdn.discordapp.com/attachments/609230525604954135/610155628786221117/unknown.png
```
Comment (Kampfkarren) [inline docs/guide/existing-game.md]: This should be an actual image, not just a link (``)

File: docs/guide/existing-game.md (hunk 1 above)
Comment (Jaguar-515) [reply docs/guide/existing-game.md]: Oh, ok. Lemme fix this.

File: docs/guide/existing-game.md (hunk 2 above)
Comment (Jaguar-515) [reply docs/guide/existing-game.md]: Alright, I'll fix it.

Comment (Jaguar-515) [reply docs/guide/existing-game.md]: Fixed. Check the latest commit.

File: docs/guide/existing-game.md (hunk 1 above)
Comment (Jaguar-515) [reply docs/guide/existing-game.md]: Fixed. Check the latest commit.

**Comment by Jaguar-515** [comment]
One question: Should I include `rojo export`?

**Comment by Jaguar-515** [comment]
Pull request is now ready for review!

**Comment by Jaguar-515** [comment]
Closed because LPG said that this guide should be in rbxlx-to-rojo README on Discord.

**Pull request closed without merging**